UnitsNet.NanoFramework/GeneratedCode/Temperature/UnitsNet.NanoFramework.Temperature.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>UnitsNet.nanoFramework.Temperature</id>
+    <version>4.89.0</version>
+    <title>Units.NET - Temperature</title>
```

Comment:
Should nanoFramework should be in the title?
Our WRC target has title `Units.NET - Windows Runtime Component`.

Maybe something like this would make the titles consistent?
`Units.NET Temperature - nanoFramework`
`Units.NET Core - nanoFramework`

Reply:
Done!